### ISSUE TITLE
log_hostname using wrong variable

### ISSUE THREAD

**Issue by gclough** [ANXS/postgresql#413]
Wrong variable

```
log_hostname = {{ 'on' if postgresql_log_duration else 'off' }}
```

**Comment by gclough** [comment]
And another:

```
krb_caseins_users = {{ 'on' if postgresql_db_user_namespace else 'off' }}
```

**Comment by gclough** [comment]
And another:

```
geqo = {{'on' if postgresql_enable_tidscan else 'off'}}
```

**Comment by gclough** [comment]
FIxed by #415

**Comment by gclough** [comment]
Fixed.
